arviz/plots/dotplot.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dot(
+    values=None,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="C0",
+    intervalcolor="C3",
+    markersize=None,
+    markercolor="C0",
+    marker="o",
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    nquantiles=50,
+    quartiles=True,
+    point_interval=None,
+    ax=None,
+    show=None,
+    plot_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+
+    """Plot distribution as dot plot or quantile dot plot.
+
+    This function uses the Wilkinson's Algorithm
+    (Leland Wilkinson (1999) Dot Plots, The American Statistician, 53:3, 276-281,
+    DOI: 10.1080/00031305.1999.10474474)to allot dots to bins.
+    The quantile dot plots was inspired from the paper(Matthew Kay, Tara Kola, Jessica R. Hullman,
+    and Sean A. Munson. 2016. When (ish) is My Bus? User-centered Visualizations of Uncertainty in
+    Everyday, Mobile Predictive Systems. DOI:https://doi.org/10.1145/2858036.2858558).
+
+    Parameters
+    ----------
+    values : array-like
+        Values to plot
+    binwidth : float
```

Comment:
```suggestion
    binwidth : float, optional
```

and same with the rest of optional parameters